### ISSUE TITLE
run eavl_image_classifer.py  an error occurred

### ISSUE THREAD

**Issue by jimmery-XUT** [tensorflow/models#9532]
totalMemory: 7.79GiB freeMemory: 6.87GiB
2020-12-07 09:36:30.525600: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2020-12-07 09:36:30.784540: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-12-07 09:36:30.784561: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2020-12-07 09:36:30.784566: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2020-12-07 09:36:30.784638: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6609 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080, pci bus id: 0000:01:00.0, compute capability: 7.5)
INFO:tensorflow:Restoring parameters from saver/inv3_flowers/model.ckpt-10000
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
2020-12-07 09:36:33.112736: E tensorflow/stream_executor/cuda/cuda_dnn.cc:352] Could not create cudnn handle: CUDNN_STATUS_ALLOC_FAILED

Process finished with exit code 139 (interrupted by signal 11: SIGSEGV)

How to solve this problem

**Comment by ravikyram** [comment]
@jimmery-XUT 

Can you try by setting `config.gpu_options.allow_growth = True.` and see if it works

Also refer similar issues [SO link](https://stackoverflow.com/questions/43147983/could-not-create-cudnn-handle-cudnn-status-internal-error) and [issue](https://github.com/tensorflow/tensorflow/issues/39989) and see if it helps you.
If still issue persists can you [please fill issue template](https://github.com/tensorflow/models/issues/new/choose) and share related code. Thanks!

**Comment by jimmery-XUT** [comment]
thank you,I have solve it .Run train_image_classifer.py with default parameters .Run eval_image_classifer.py ,the accuracy rate is only 46%. How can I adjust the parameters similar to the learning rate to improve the accuracy.
Do you have any good suggestions？

**Comment by ravikyram** [comment]
@jimmery-XUT 

Please, close the issue as the original issue was resolved.

Please, raise the new issue by filling issue template and related code, to debug the new  issue reported here. Thanks!

**Comment by ravikyram** [comment]
@jimmery-XUT 

Can we close this issue here and track the issue in #9534. Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
